hoomd/filter/filter_.py

```diff
@@ -1,21 +1,32 @@
-from hoomd._hoomd import ParticleFilter
+"""Define ParticleFilter."""
 
+from hoomd import _hoomd
 
-class _ParticleFilter(ParticleFilter):
-    """Base class for all particle filters.
 
-    Should not be instantiated or subclassed by users.
-    """
+class ParticleFilter(_hoomd.ParticleFilter):
+    """Base class for all particle filters."""
```

Comment:
I think we should still document that users should not create instances of this class. If they do and use it as a particle filter, it won't break right away. It will only break when attaching and raise a pybind11 error about the constructor of `_hoomd.ParticleFilter` from what I can glean from the code.

Reply:
Done.